CRM/Queue/DAO/Queue.php

```diff
@@ -56,13 +56,49 @@ class CRM_Queue_DAO_Queue extends CRM_Core_DAO {
   public $type;
 
   /**
-   * Should the standard background attempt to autorun tasks in this queue?
+   * Name of the task runner
    *
-   * @var bool|string|null
-   *   (SQL type: tinyint)
+   * @var string
+   *   (SQL type: varchar(64))
    *   Note that values will be retrieved from the database as a string.
    */
-  public $is_autorun;
+  public $runner;
+
+  /**
+   * Maximum number of items in a batch. Tip: If you expand batch_limit, then also consider expanding lease_time.
```

Comment:
I don't think it's important but I found this 'tip' confusing/ better left off 

First I didn't understand what 'expand' meant in this context & then when I looked the batch limit is already an hour so it is unlikely to be tightly coupled to the batch limit - ie you could hit an hour on 1 item as easily as 10,000 from my experience (e.g 1 db update with a cross join on a medium sized db would usually take longer than importing 20,000 contacts)

Reply:
OK, I can trim that.

The relation should be like `lease_time > expected_time_per_item * batch_limit`.

I think there will be some onus on the devdocs to use more realistic/attuned numbers. The 1 hour is pretty arbitrary IMHO.